### ISSUE TITLE
[fable-standalone] minor issues

### ISSUE THREAD

**Issue by ncave** [fable-compiler/Fable#1742]
Some failing tests when compiled with `fable-standalone`:
```
  isNaN (sqrt System.Double.NaN) |> equal true
  SByte.Parse("-128") |> equal -128y
  Int16.Parse("-32768") |> equal -32768s
```
Compiles (incorrectly) to:
```
  equal(true, isNaN(Math.sqrt(null))); // <-- null is incorrect
  equal(128, (0, _Int.parse)("-128", 511, false, 8)); // <-- 128 is incorrect
  equal(32768, (0, _Int.parse)("-32768", 511, false, 16)); // <-- 32768 is incorrect
```
I'll take a look.

**Comment by ncave** [comment]
The double.NaN serialization fixed by #1743.

**Comment by ncave** [comment]
The integer issue above turns out to be an issue with unary minus on INT_MIN:
in F#:  -(-128y) = -128y; -(-32768s) = -32768; -(-2147483648) = -2147483648
in Fable: -(-128y) = 128; -(-32768s) = 32768; -(-2147483648) = 2147483648

**Comment by ncave** [comment]
Fixed in #1743.

**Comment by ncave** [comment]
Closing as fixed.
